README.md

```diff
@@ -54,17 +54,32 @@ OpenPGP.js [![Build Status](https://travis-ci.org/openpgpjs/openpgpjs.svg?branch
     | p384            | ECDH       | ECDSA     | Yes      | Yes*       | Yes*      |
     | p521            | ECDH       | ECDSA     | Yes      | Yes*       | Yes*      |
     | secp256k1       | ECDH       | ECDSA     | Yes      | Yes*       | No        |
-    | curve25519      | ECDH       | N/A       | Yes      | No (TODO)  | No        |
-    | ed25519         | N/A        | EdDSA     | Yes      | No (TODO)  | No        |
-    | brainpoolP256r1 | ECDH       | ECDSA     | Yes      | No (TODO)  | No        |
-    | brainpoolP384r1 | ECDH       | ECDSA     | Yes      | No (TODO)  | No        |
-    | brainpoolP512r1 | ECDH       | ECDSA     | Yes      | No (TODO)  | No        |
+    | brainpoolP256r1 | ECDH       | ECDSA     | Yes      | Yes*       | No        |
+    | brainpoolP384r1 | ECDH       | ECDSA     | Yes      | Yes*       | No        |
+    | brainpoolP512r1 | ECDH       | ECDSA     | Yes      | Yes*       | No        |
+    | curve25519      | ECDH       | N/A       | Yes      | No         | No        |
+    | ed25519         | N/A        | EdDSA     | Yes      | No         | No        |
```

Comment:
Just out of curiosity, is there a specific reason why X25519 with Node Crypto doesn't work?

Reply:
Yeah, see https://github.com/openpgpjs/openpgpjs/pull/680#issuecomment-376439537